content/docs/guides/crosswalk/aws/iam.md

```diff
@@ -109,7 +109,7 @@ const rolePolicyAttachment = new aws.iam.RolePolicyAttachment("rpa", {
 ```
 
 For a full list of available managed policy ARNs, please refer to the
-[API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam" >}}).
+[API documentation]({{< relref "/docs/reference/pkg/aws/iam" >}}).
```

Comment:
I am a little conflicted with the links we should have on this page. On the one hand having them link to the nodejs API docs (as they are today) makes sense because all of the code snippets are in TS. But I also like that we can link to a specific resource such as User, AccessKey UserPolicyAttachment under the IAM module, which lets the user see just the relevant resource. I am fine with keeping the changes you've made here but would like to see what @justinvp of this as well.

Reply:
Yeah, I'm a bit conflicted as well, but I do think the resource docs are a better representation of what we want to display to users first, and then they can click the link to the specific language API from there, so I'm fine with linking to those. Eventually these docs should be updated for more languages anyways (as the multi-lang effort progresses), so we'll eventually be linking to the resource docs first anyway.